### PR TITLE
Release/0.6.0

### PR DESCRIPTION
**Pull request by santtul**
Release 0.6.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,107 +3,91 @@
   "version": "0.5.2",
```
Comment (Nizar-Rahme) [inline package.json:3]: Bumping up the version number here is not only a matter of following best practices but also has some logic tied to it for flushing the cache from the users' browsers.

**Review by Nizar-Rahme — CHANGES_REQUESTED** [review]
Lots of changes have been introduced in this version. It would be great if we keep maintaining `CHANGELOG.md`

File: package.json (hunk 1 above)
Comment (santtul) [reply package.json:3]: would do these in another PR if it's ok?

Comment (igordavydsson) [reply package.json:3]: yeah, version bump should be a separate PR IMO

Comment (igordavydsson) [reply package.json:3]: release PR only for checking that all the commits about to be released are OK

**Review by Nizar-Rahme — APPROVED** [review]
As bumping up the version number would be done in another PR, this looks good 👍

**Pull request merged**